canvas_modules/common-canvas/src/common-properties/components/control-item/control-item.jsx

```diff
@@ -99,7 +99,11 @@ class ControlItem extends React.Component {
 
 		const action = this.actionFactory.generateAction(0, this.props.control.action);
 
-		const className = classNames("properties-control-item", { "hide": hidden }, { "properties-ci-action-item": action });
+		const className = classNames(
+			"properties-control-item",
+			{ "hide": hidden },
+			{ "properties-ci-action-item": action && this.props.control.action.actionType === ActionType.IMAGE }
```

Comment:
This class was added to fix alignment issue for Action image in this PR - https://github.com/elyra-ai/canvas/pull/2258

But it was being applied to Action button and custom actions as well. Fixed the condition to add this class for Action image only.